### DISCUSSION TITLE
Github Access Token Required Scopes

### DISCUSSION THREAD

**Discussion by ndoell** [projectdiscovery/subfinder#554] [Q&A] [answered]
Hi,

When provisioning a GitHub token to be used in the provider-config.yml what are the minimum scopes that need to be applied?

Thank you

**Comment by ehsandeep** [comment, marked as answer]
>  public_repo Access public repositories

<img width="408" alt="image" src="https://user-images.githubusercontent.com/8293321/172420371-487e0c87-eac8-469e-9fd1-aff312828805.png">

**Reply by ndoell** [reply]
Thank you!
